src/modules/interact.js

```diff
@@ -0,0 +1,23 @@
+/* eslint-disable operator-linebreak */
+export const baseURL =
```

Comment:
- [ ] As requested by the previous code reviewer, Please do not disable any linter checks as linters are here to help us write readable and professional-looking codes. Disabling the linters shows that you do not follow the Microverse project standards. Kindly fix all linter errors instead of disabling them so as to meet all the project requirements. You can read about the error and how to fix it [here](https://eslint.org/docs/rules/operator-linebreak)